web/documentserver-example/nodejs/app.js

```diff
@@ -651,6 +651,22 @@ app.get("/editor", function (req, res) {
     }
 });
 
+app.get("/files", function(req, res){
+    try {
+        docManager.init(storageFolder, req, res); 
+        const filesInFolderInfo = docManager.getFilesInFolderInfo();
+        const responseObj = {
+            count: filesInFolderInfo.length,
+            response: filesInFolderInfo
+        };
+        res.write(JSON.stringify(responseObj));
+    } catch (ex){
+        console.log(ex);
+        res.write("Server error");
+    }
+    res.end();
+})
```

Comment:
Please move this code to the file request level
https://github.com/ONLYOFFICE/document-server-integration/blob/48c0bef1858b55184ea8128ede86198df52d0592/web/documentserver-example/nodejs/app.js#L291

Reply:
I saw it, but did not know what to call it correctly